### ISSUE TITLE
Buttons are not having discernible text.

### ISSUE THREAD

**Issue by msftedad** [sag1v/react-elastic-carousel#255]
**Describe the bug**
Element does not have inner text that is visible to screen readers
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute
Element's default semantics were not overridden with role="none" or role="presentation"

**To Reproduce**
1.Open the Sample app.
2. Open the Accessibility Insights tool for web.
3. Run the fast pass and verify whether the tool is capturing the issue "Ensures buttons have discernible text" or not?

**Expected behavior**
Buttons should have discernible text.

**Sandbox link**
https://codesandbox.io/s/react-carousel-a11y-bug-fzov3q

**Screenshots**
![210058185-1c8b5f7a-b5b2-4639-bef5-71ccf0375080](https://user-images.githubusercontent.com/93735775/211783354-7b2bb83e-7c7f-43b5-8ee6-5a65bf42f854.png)

**Desktop (please complete the following information):**
 - OS: 22621.963
 - Browser Microsoft Edge
 - Version 108.0.1462.76 (Official build) (64-bit)

**Additional context**
N/A

**Comment by msftedad** [comment]
Hi Team,
Thanks for reaching us and still we are able to reproduce this issue and please let us know if anything need more information from our side.

**Comment by msftedad** [comment]
Hi Team,
Thanks for reaching us and still we are able to reproduce this issue and please let us know if anything need more information from our side. Please can anyone work on this and let us know the update.
